spec/core_functions/color/mix/named.hrx

```diff
@@ -0,0 +1,27 @@
+<===> options.yml
+:ignore_for:
+  - libsass
+
+<===>
+================================================================================
+<===> polar_space/options.yml
+:todo:
+  - dart-sass
+
+<===> polar_space/input.scss
+a {b: mix($color1: #91e16f, $color2: #0144bf, $weight: 92%, $method: hsl decreasing hue)}
+
+<===> polar_space/output.css
+a {
+  b: rgb(170.1337620418, 206.8414431786, 78.2787893421);
```

Comment:
This is quite different than Dart Sass's current output of `rgb(177.749777646, 225.4953896552, 98.9846103448)`. Which is right?

Reply:
The Dart Sass output is correct. I've updated with the correct output here.